src/web3/saga.ts

```diff
@@ -84,7 +92,19 @@ export function* getOrCreateAccount() {
       throw new Error('Failed to generate mnemonic')
     }
 
-    const keys = yield call(generateKeys, mnemonic, undefined, undefined, undefined, bip39)
+    const DERIVATION_PATH = useTwelveWordSeedPhrase
+      ? ETHEREUM_DERIVATION_PATH
+      : CELO_DERIVATION_PATH_BASE
+
+    const keys = yield call(
```

Comment:
could we use `generateKeysFromMnemonic` instead? that would remove the need to specify the derivation path here

Reply:
Done in 67c68573612d31507eca1d95de789002ea28ea2f!